src/platform/Infineon/crypto/trustm/CHIPCryptoPALHsm_P256_trustm.cpp

```diff
@@ -136,11 +123,20 @@ CHIP_ERROR P256Keypair::Initialize(ECPKeyTarget key_target)
     }
     else
     {
+#if !ENABLE_TRUSTM_NOC_KEYGEN
+        if (CHIP_NO_ERROR == Initialize_H(this, &mPublicKey, &mKeypair))
+        {
+            mInitialized = true;
+        }
+        error = CHIP_NO_ERROR;
```

Comment:
Why does this ignore errors from `Initialize_H` ?

Reply:
Updated this part.